stdlib/src/array.rs

```diff
@@ -581,7 +581,7 @@ mod array {
     }
 
     #[pyattr]
-    #[pyclass(name = "array")]
+    #[pyclass(name = "array.array")]
```

Comment:
we typically put only `array` here. the other array came from the module name.

Reply:
Understood! I fixed it and added tests.